public/static/docs/user-guide/analytics.md

```diff
@@ -25,9 +25,8 @@ User and event data have a 14 month retention period.
 
 DVC's analytics record the following information per event:
 
-- The DVC version, e.g. `0.22.0`
-- The operating system information, e.g. `linux`, `ubuntu`, `14.04`, etc
-- The underlying version control system, e.g. `git`
+- The DVC version, e.g. `0.8.0`
```

Comment:
0.82?

Reply:
True. It's just an example but updated, thanks.